ui/helpers/utils/transactions.util.js

```diff
@@ -71,18 +62,11 @@ export async function getMethodDataAsync(fourBytePrefix) {
       registry = new MethodRegistry({ provider: global.ethereumProvider });
     }
 
-    const registrySig = await registry.lookup(fourBytePrefix).catch((e) => {
-      log.error(e);
-      return null;
-    });
-
-    const sig = pickShortest(registrySig, fourByteSig);
-
-    if (!sig) {
+    if (!fourByteSig) {
       return {};
     }
 
-    const parsedResult = registry.parse(sig);
+    const parsedResult = registry.parse(fourByteSig);
```

Comment:
You can remove part of the `nock` fixtures for this test now too.

Reply:
removed [here](https://github.com/MetaMask/metamask-extension/pull/15300/commits/13b6b30eec0a6a7ac02d8ea968c932ced94d531f)